.github/workflows/maven.yml

```diff
@@ -28,7 +28,10 @@ jobs:
           distribution: 'temurin'
           cache: maven
       - name: Build with Maven and run linting checks
-      env:
+        env:
+          BASE_URL: ${{ vars.BASE_URL }}
```

Comment:
If these are required to be passed to the application via docker then the `dockerfile` also needs updates

Reply:
Fixed :)